### PR TITLE
MWPW-132671: Load Milo's locales for config

### PR DESCRIPTION
**Pull request by hparra**
Supports: [MWPW-132671](https://jira.corp.adobe.com/browse/MWPW-132671)

Requires: https://github.com/adobecom/milo/pull/880

**Test URLs:**

- Before: https://use-milo-locales--dc--adobecom.hlx.live/ar/acrobat/online/word-to-pdf
- After: https://use-milo-locales--dc--adobecom.hlx.live/ar/acrobat/online/word-to-pdf?milolibs=import-locales--milo--hparra

### CODE REVIEW

File: acrobat/scripts/scripts.js [hunk 1]
```diff
@@ -201,12 +99,13 @@ const CONFIG = {
   loadStyles(paths);
 
   // Import base milo features and run them
+  const { locales } = await import(`${miloLibs}/scripts/scripts.js`);
```
Comment (peller) [inline acrobat/scripts/scripts.js:102]: this will make our page load slower.

**Review by peller — CHANGES_REQUESTED** [review]
let's discuss performance ramifications of this change

File: acrobat/scripts/scripts.js (hunk 1 above)
Comment (hparra) [reply acrobat/scripts/scripts.js:102]: Change is in the microseconds. Will be loaded from `/scripts/locales.js` when change is made in Milo. This is POC right now.

Comment (peller) [reply acrobat/scripts/scripts.js:102]: everything is microseconds.  hundreds of microseconds on a constrained connection.  If this is merged, won't it go live?

File: acrobat/scripts/scripts.js [hunk 2]
```diff
@@ -201,12 +99,17 @@ const CONFIG = {
   loadStyles(paths);
 
   // Import base milo features and run them
-  const {
-    loadArea, loadDelayed, loadScript, setConfig, loadLana, getMetadata, getLocale
-  } = await import(`${miloLibs}/utils/utils.js`);
+  const [
+    { locales },
+    { loadArea, loadDelayed, loadScript, setConfig, loadLana, getMetadata, getLocale }
+  ] = await Promise.all([
```
Comment (peller) [inline acrobat/scripts/scripts.js:105]: Doing this in parallel is better, but everything that comes before loadArea is critical, so we had inlined this stuff on purpose (even though duplicating code is not the ideal approach)

What is the motivation for changing this?

@Blainegunn

Comment (peller) [reply acrobat/scripts/scripts.js:105]: ...ideally we'd have a rollup/webpack builder process our imports and do this for us.

**Comment by hparra** [comment]
@peller See https://github.com/adobecom/milo/pull/880 and the issue it links to for context. I'll recreate this PR against staging if/when that lands.

**Pull request closed without merging**